packages/workspace/src/node/default-workspace-server.spec.ts

```diff
@@ -0,0 +1,109 @@
+/********************************************************************************
+ * Copyright (C) 2022.
```

Comment:
The copyright header looks malformed, you'll need to either reference the company you work for, or your name.

Reply:
adapted :thumbsup: